app/views/logix/index.html.erb

```diff
@@ -1,3 +1,12 @@
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.0/jquery.min.js"></script>
```

Comment:
Did you check if jQuery is already included in the application. If it is already included, then you don't need to include it here. Please recheck this.

Everything else is good to go. 👏